app/Http/Requests/DestroyPageRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+use App\Models\Role;
+use Auth;
+
+class DestroyPageRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        $user = Auth::user();
+        return $user && $user->hasRole(Role::ROLE_ADMIN);
```

Comment:
Why can't you create an AuthorizedRequest class that extends Request and defines authorize() and use that as a base for all of these other Request objects that use it?  Seems silly to copy and paste that code over and over.